dragonfly/block/model/lantern.go

```diff
@@ -0,0 +1,17 @@
+package model
+
+import (
+	"github.com/df-mc/dragonfly/dragonfly/entity/physics"
+	"github.com/df-mc/dragonfly/dragonfly/world"
+	"github.com/go-gl/mathgl/mgl64"
+)
+
+type Lantern struct{}
+
+func (l Lantern) AABB(pos world.BlockPos, w *world.World) []physics.AABB {
+	return []physics.AABB{physics.NewAABB(mgl64.Vec3{}, mgl64.Vec3{0.375, 0.5, 0.375})}
```

Comment:
I think the AABB is different depending on if the lantern is hanging or not too. Make sure to take that into account.